Gemfile

```
@@ -1,15 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'oj'
-gem 'htmlentities'
-gem 'net-dns'
-gem 'bit-struct'
-gem 'geoip-c'
-gem 'maxmind-db', '~> 1.0.0'
-gem 'recog', '~> 3.0'
+gemspec name: 'dap'
```

Was there a reason to name this?  Since there is only one gemspec in this repo seems reasonable to just leave that off.
```suggestion
gemspec
```

---

I was simply following a pattern I saw elsewhere, and I now realize is inappropriate unless there were multiple gemspecs in the same directory.